app/components/file-browser/component.js

```diff
@@ -62,7 +62,7 @@ export default Ember.Component.extend(Analytics, {
     },
     currentUser: Ember.inject.service(),
     dropzone: Ember.computed.alias('edit'),
-    edit: Ember.computed('user', function() {
+    edit: Ember.computed('user', 'currentUser.currentUserId', function() {
```

Comment:
For maximum correctness, it should probably depend on `user.id`, not just `user`. But that's not a big deal.